cypress/e2e/groups/RenameGroupModal.cy.js

```diff
@@ -0,0 +1,22 @@
+const dayjs = require('dayjs')
+
+describe('Create group modal', () => {
+  before(() => {
+    cy.beforeTest('/fleet-management')    
+  });
+
+  it('create group modal happy path', function () {  
```

Comment:
This is the rename group test, but L3 and L8 indicate it's a create group test.

Reply:
thanks! i didn't notice, i just  pushed the fix